test/models/request/ArrivalNotificationRequestSpec.scala

```diff
@@ -45,17 +46,20 @@ class ArrivalNotificationRequestSpec
     with ScalaCheckDrivenPropertyChecks
     with OptionValues {
 
-  private val dateTime: LocalDateTime = LocalDateTime.now()
-  private val dateOfPreparation       = Format.dateFormatted(dateTime)
-  private val timeOfPreparation       = Format.timeFormatted(dateTime)
+  private val dateOfPreparation          = LocalDate.now()
+  private val dateOfPreparationFormatted = Format.dateFormatted(dateOfPreparation)
+  private val timeOfPreparation          = LocalTime.now()
+  private val timeOfPreparationFormatted = Format.timeFormatted(timeOfPreparation)
```

Comment:
^^ Ditto ^^

Reply:
`timeOfPreparation` is used below to build the expect statement